Installation/lib/cmake/CGAL/CGALConfig.cmake

```diff
@@ -35,6 +35,8 @@ if(CGALConfig_all_targets_are_defined)
   return()
 endif()
 
+include(${CGAL_MODULES_DIR}/CGAL_CreateSingleSourceCGALProgram.cmake)
+
```

Comment:
That is not sufficient. Travis still fails:
https://travis-ci.org/CGAL/cgal/jobs/443562443
```
CMake Error at /usr/local/lib/cmake/CGAL/CGALConfig.cmake:38 (include):
  include could not find load file:
    /CGAL_CreateSingleSourceCGALProgram.cmake
```

Reply:
Any reason why I don't have this error locally ?

Reply:
That `CGALConfig.cmake` is the one that is installed by `make install` (after a configuration with `CGAL_HEADER_ONLY`). It might be different from the one you use.